measure_cocoapod_size.py

```diff
@@ -96,6 +106,18 @@ def CopyProject(source_dir, target_dir):
   """
   os.system('cp -r {} {}'.format(source_dir, target_dir))
 
+def ValidateSourceConfig(pod_sources):
+  for sdk , source in pod_sources.items():
+    if len(source) >= 1 and ( source.keys()[0] not in set(["git", "path"]) ):
+      raise ValueError(
+              "Pod source of SDK {} should be `git` or `path`.".format(sdk))
+    elif len(source) > 1:
```

Comment:
`elif len(source) == 2` seems more explicit in this case. The config dict should two entries at most (i.e. git => tag =>) right?